pypika/tests/clickhouse/test_search_string.py

```diff
@@ -1,33 +1,33 @@
 import unittest
 
+from parameterized import parameterized
+
 from pypika import Field
 from pypika.clickhouse.search_string import Match, Like, NotLike, MultiSearchAny, MultiMatchAny
 
 
 class TestSearchString(unittest.TestCase):
-    def test_search_string(self):
-        search_types = (
-            (Match, "match(toString(\"name\"),'twheys')"),
-            (Like, "like(toString(\"name\"),'twheys')"),
-            (NotLike, "notLike(toString(\"name\"),'twheys')"),
-        )
 
-        for cls, expected in search_types:
-            self.assertEqual(
-                cls(Field('name'), 'twheys'),
-                expected
-            )
+    @parameterized.expand([
```

Comment:
btw this library is a really good addition for the types of tests we write in pypika. Appreciate you adding it.